docs/source/external_resources.rst

```diff
@@ -2,6 +2,13 @@
 External Resources
 ==================
 
+1. stanford flightroom wiki
```

Comment:
is this page going to change? I feel like my recipes are being stolen from me without compensation.

Reply:
leftover from prior fork. can ignore.